src/account/account.js

```diff
@@ -4,6 +4,7 @@ const eosECC = require('eosjs').modules.ecc;
 class Account {
 
     constructor(name, privateKey, provider) {
+        this.keys = {}
```

Comment:
What is the point of `keys` object ? All of the blockchain actions behind the scene are executed from the active key and in that case when you want to load a blockchain account you will provide only the active key. Could you give me a use case ?